### PR TITLE
Material_AlphaBlend refactor

### PR DESCRIPTION
**Pull request by stevk**
Refactor for Material_AlphaBlend

### CODE REVIEW

File: Output/Manifest.json [hunk 1]
```diff
@@ -91,5 +91,87 @@
         }
       }
     ]
+  },
```
Comment (kcoley) [inline Output/Manifest.json]: Is this Manifest file missing models?

File: Output/Material_AlphaBlend/Material_AlphaBlend_00.gltf [hunk 2]
```diff
@@ -79,6 +76,7 @@
   ],
   "materials": [
     {
+      "pbrMetallicRoughness": {},
```
Comment (kcoley) [inline Output/Material_AlphaBlend/Material_AlphaBlend_00.gltf]: Can this line be removed from export?

File: Output/Material_AlphaBlend/README.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ The following table shows the properties that are set for a given model.
...
-| [05](Material_AlphaBlend_05.gltf)<br>[View](https://bghgary.github.io/glTF-Assets-Viewer/?folder=3&model=5) | [<img src="Figures/Thumbnails/Material_AlphaBlend_05.png" align="middle">](Figures/SampleImages/Material_AlphaBlend_05.png) |   | [<img src="Figures/Thumbnails/BaseColor_Plane.png" align="middle">](Textures/BaseColor_Plane.png) | [1.0,&nbsp;1.0,&nbsp;1.0,&nbsp;0.7] |
-| [06](Material_AlphaBlend_06.gltf)<br>[View](https://bghgary.github.io/glTF-Assets-Viewer/?folder=3&model=6) | [<img src="Figures/Thumbnails/Material_AlphaBlend_06.png" align="middle">](Figures/SampleImages/Material_AlphaBlend_06.png) | Vector4 Float | [<img src="Figures/Thumbnails/BaseColor_Plane.png" align="middle">](Textures/BaseColor_Plane.png) | [1.0,&nbsp;1.0,&nbsp;1.0,&nbsp;0.7] |
+| [00](Material_AlphaBlend_00.gltf)<br>[View](https://bghgary.github.io/glTF-Assets-Viewer/?folder=1&model=0) | [<img src="Figures/Thumbnails/Material_AlphaBlend_00.png" align="middle">](Figures/SampleImages/Material_AlphaBlend_00.png) | Vec 4 Float |   |   |
```
Comment (kcoley) [inline Output/Material_AlphaBlend/README.md]: Has the folder order changed?

File: README.md [hunk 4]
```diff
@@ -7,6 +7,7 @@ In order to help ensure that glTF 2.0 importers meet the [glTF 2.0 Specification
 ## What does it currently cover?
 So far, this project covers these requirements, with more to come:
 - [Material](Output/Material/README.md)
+- [Material Alpha Blend](Output/Material_AlphaBlend/README.md)
```
Comment (kcoley) [inline README.md]: Are these models missing on purpose?

File: Output/Material_AlphaBlend/Material_AlphaBlend_00.gltf (hunk 2 above)
Comment (stevk) [reply Output/Material_AlphaBlend/Material_AlphaBlend_00.gltf]: Fixed

**Pull request merged**